From 7deeb48b7d03561224c8442d946a1acbc603d096 Mon Sep 17 00:00:00 2001
From: Morgan Parry <redacted@redacted.com>
Date: Fri, 11 Jul 2025 12:41:20 +0100
Subject: [PATCH] relax protobuf

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index db4b96d..1ada945 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,7 +26,7 @@ classifiers = [
   "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-  "protobuf>=5.0, < 6.0",
+  "protobuf>=5.0, < 7.0",
 ]
 
 [project.urls]
-- 
2.42.0.windows.2

